include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -1323,6 +1323,29 @@ namespace internal
 
 
 
+      /**
+       * Returns all active fe indices on a given vertex.
+       *
+       * The size of the returned set equals the number of finite elements that
+       * are active on this vertex.
+       */
+      template <int dim, int spacedim>
+      static std::set<unsigned int>
+      get_active_vertex_fe_indices(
```

Comment:
I actually need the function `get_active_fe_indices` first in the succeeding pull request, but I thought it would fit nicely in this pull request since we introduce the corresponding function for  `TriaObjects` here.

I don't think I actually need a template specialization for `get_active_fe_indices` in `DoFAccessor<0,spacedim>`, since it already calls specialized member functions internally, which trigger the corresponding assertions. I could provide a specialization though if you want.

Reply:
If you promise that you'll use the function in a follow-up PR, then let's leave it in. In general, adding code that is not used anywhere is a prescription for creating dead code that eventually bit-rots and doesn't work any more -- so we like to not do it.

But if you're about to use it, then that's ok with me.